### PR TITLE
Random list shuffle

### PR DESCRIPTION
**Pull request by ilyakaznacheev**
- shuffle slices and get the next element from the slice instead of getting a random element each time - will guarantee a 100% message usage
- tests

### CODE REVIEW

**Comment by ilyakaznacheev** [comment]
@achikin

File: internal/message/message.go [hunk 1]
```diff
@@ -7,11 +7,44 @@ import (
...
+// it will begin from the first element if the counter will reach the end of the slice
+func (s *ShuffleStringSlice) GetNext() string {
+	if len(s.s) == 0 {
+		return ""
+	}
+
+	s.i++
+	if s.i == len(s.s) {
+		s.i = 0
+	}
```
Comment (khorpy) [inline internal/message/message.go:37]: We need to reshuffle the array when reaching the end, or we'd constantly get the same sequence of phrases.

**Review by khorpy — CHANGES_REQUESTED** [review]

File: internal/message/message.go (hunk 1 above)
Comment (ilyakaznacheev) [reply internal/message/message.go:37]: Makes sense

Comment (ilyakaznacheev) [reply internal/message/message.go:37]: Done

**Review by khorpy — DISMISSED** [review]

**Review by aikikode — APPROVED** [review]

**Pull request merged**